test/integration/controllers/admin.spec.js

```diff
@@ -0,0 +1,66 @@
+const { serverInit, serverCleanup } = require('~/test/setup')
+const {
+  roles: { ADMIN }
+} = require('~/consts/auth')
+const User = require('~/models/user')
+const Role = require('~/models/role')
+const { USER_NOT_REGISTERED } = require('~/consts/errors')
+
+describe('Admin controller', () => {
+  let app, server
+
+  beforeAll(async () => {
+    ;({ app, server } = await serverInit())
+  })
+
+  afterAll(async () => {
+    await serverCleanup(server)
+  })
+
+  it('getAdmins', async () => {
+    const response = await app.get('/admins')
+
+    expect(response.statusCode).toBe(200)
+    expect(response.body).toEqual([])
+  })
+
+  describe('get by id', () => {
+    it('found', async () => {
+      const role = await Role.findOne({ value: ADMIN }).lean().exec()
+      let adminToSave = {
+        role,
+        firstName: 'John',
+        lastName: 'Doe',
+        email: 'john@example.com',
+        password: 'password'
+      }
+
+      const admin = await User.create(adminToSave)
+
+      const response = await app.get(`/admins/${admin._id}`)
+
+      const { id, firstName, lastName, email } = response.body
+
+      expect(response.statusCode).toBe(200)
+      expect(id).toBe(admin._id.toString())
```

Comment:
Please use `expect.objectContaining()` method to check whether `response.body` has `adminToSave` as a subset (you can exclude password, and role), e. g. `expect(response.body).toEqual(expect.objectContaining(adminToCheck))`

I think it'll be better than checking for all the fields.